Documentation/bpf/progtypes.rst

```diff
@@ -338,49 +338,88 @@ corresponding network driver name of an interface can be determined as follows:
-| Cavium            | thunderx   | >= 4.12     |
-+-------------------+------------+-------------+
-| Freescale         | dpaa2      | >= 5.0      |
-+-------------------+------------+-------------+
-| Intel             | ixgbe      | >= 4.12     |
-|                   +------------+-------------+
-|                   | ixgbevf    | >= 4.17     |
-|                   +------------+-------------+
-|                   | i40e       | >= 4.13     |
-|                   +------------+-------------+
-|                   | ice        | >= 5.5      |
-+-------------------+------------+-------------+
-| Marvell           | mvneta     | >= 5.5      |
-+-------------------+------------+-------------+
-| Mellanox          | mlx4       | >= 4.8      |
-|                   +------------+-------------+
-|                   | mlx5       | >= 4.9      |
-+-------------------+------------+-------------+
-| Microsoft         | hv_netvsc  | >= 5.6      |
-+-------------------+------------+-------------+
-| Netronome         | nfp        | >= 4.10     |
-+-------------------+------------+-------------+
-| Others            | virtio_net | >= 4.10     |
-|                   +------------+-------------+
-|                   | tun/tap    | >= 4.14     |
-|                   +------------+-------------+
-|                   | bond       | >= 5.15     |
-+-------------------+------------+-------------+
-| Qlogic            | qede       | >= 4.10     |
-+-------------------+------------+-------------+
-| Socionext         | netsec     | >= 5.3      |
-+-------------------+------------+-------------+
-| Solarflare        | sfc        | >= 5.5      |
-+-------------------+------------+-------------+
-| Texas Instruments | cpsw       | >= 5.3      |
-+-------------------+------------+-------------+
++------------------------+-------------------------+-------------+
+| Vendor                 | Driver                  | XDP Support |
++========================+=========================+=============+
+| Amazon                 | ena                     | >= 5.6      |
++------------------------+-------------------------+-------------+
+| Aquantia               | atlantic                | >= 5.13     |
++------------------------+-------------------------+-------------+
+| Broadcom               | bnxt_en                 | >= 4.11     |
++------------------------+-------------------------+-------------+
+| Cavium                 | thunderx                | >= 4.12     |
++------------------------+-------------------------+-------------+
+| Engleder               | tsne                    | >= 6.3      |
+|                        | (TSN Express Path)      |             |
++------------------------+-------------------------+-------------+
+| Freescale              | dpaa                    | >= 5.0      |
+|                        +-------------------------+-------------+
+|                        | dpaa2                   | >= 5.11     |
```

Comment:
The versions are reversed dpaa2 got support in 5.0 [see](https://github.com/torvalds/linux/commit/7e273a8ebdd3b83f94eb8b49fc8ee61464f47cc2)
and dpaa got support in v5.11 [see](https://github.com/torvalds/linux/commit/86c0c196cbe48f844721783d9162e46bc35c0c5a)